hw14/index.js

```diff
@@ -0,0 +1,160 @@
+const input1 = document.querySelector(".input1");
+const display1 = document.querySelector(".display1");
+const button1 = document.querySelector(".enter1");
+
+const input2 = document.querySelector(".input2");
+const display2 = document.querySelector(".display2");
+const button2 = document.querySelector(".enter2");
+
+const input3 = document.querySelector(".input3");
+const display3 = document.querySelector(".display3");
+const button3 = document.querySelector(".enter3");
+
+const input4 = document.querySelector(".input4");
+const display4 = document.querySelector(".display4");
+const button4 = document.querySelector(".enter4");
+
+const input5 = document.querySelector(".input5");
+const display5 = document.querySelector(".display5");
+const button5 = document.querySelector(".enter5");
+
+// task1
+const serchCentury = function () {
+  const century = Math.floor(+input1.value / 100) + 1;
+  let end;
+  const array = [11, 12, 13, 14];
+  if (array.includes(century)) {
+    end = "th";
+  } else if (century % 10 === 1) {
+    end = "st";
+  } else if (century % 10 === 2) {
+    end = "nd";
+  } else if (century % 10 === 3) {
+    end = "rd";
+  } else {
+    end = "th";
+  }
+  display1.innerHTML =
+    +input1.value < 10000 ? `${century}${end}` : "You kept too many numbers!";
+  input1.value = "";
+};
+button1.addEventListener("click", () => {
+  serchCentury();
+});
+
+// task 2
+let error2 = false;
+const getStringForVowelOne = function () {
+  const string = input2.value;
+  if (string.length === 0) {
+    error2 = true;
+  }
+  return string;
+};
+const vowelOne = function (string) {
+  const stringArray = string.toLowerCase().split("");
+  let result = "";
+  const arrayOfVowels = ["a", "e", "i", "j", "o", "q", "u", "y"];
+  stringArray.forEach((element) => {
+    if (arrayOfVowels.includes(element)) {
+      result += "1";
+    } else {
+      result += "0";
+    }
+  });
+  display2.innerHTML = error2 ? "You did not lead any symbol!" : result;
+  input2.value = "";
+  error2 = false;
+};
+button2.addEventListener("click", () => {
+  const string = getStringForVowelOne();
+  vowelOne(string);
+});
+
+// task 3
+let error3 = false;
+const getStringForSpinWords = function () {
+  const string = input3.value;
+  if (string.length === 0) {
+    error3 = true;
+  }
+  return string;
+};
+const spinWords = (string) => {
+  const result = string
+    .split(" ")
+    .map((element) => {
+      if (element.length > 5) {
+        return element.split("").reverse().join("");
+      }
+      return element;
+    })
+    .join(" ");
+  display3.innerHTML = error3 ? "You have not entered any words!" : result;
+  input3.value = "";
+  error3 = false;
+};
+button3.addEventListener("click", () => {
+  const strings = getStringForSpinWords();
+  spinWords(strings);
+});
+
+// task 4
+let error4 = false;
+const getHighAndLowNumber = function () {
```

Comment:
It should have name as in task, but yours sounds better)
```suggestion
const highAndLow = function () {
```